web/modules/custom/vote_utility/translations/nv.po

```diff
@@ -140,7 +140,7 @@ msgstr "I'ii'n77[ biniiy4 yah ajigh11hgo ha'dilne' bee nihoolzh77sh"
 
 # data.translations.register.dates__heading
 msgid "Voter registration deadlines"
-msgstr "I'ii'n77[ biniiy4 ha'dilne' bee nihoolzh77sh @state_name"
+msgstr "E'et'11d b1 ha'diln4h7g77 bee e'e'aah g0ne'"
 
 # data.translations.register.confirm_registration__heading
 msgid "How to check your voter registration"
```

Comment:
Remove @state_name from this string (heading 3)

Reply:
Updated in latest commit.